packages/Popover/src/Popover.js

```diff
@@ -104,6 +104,7 @@ class Popover extends React.Component {
       onDelayedClose: this.handleDelayedClose,
       onDelayedOpen: this.handleDelayedOpen,
       onOpen: this.handleOpen,
+      onAfterOpen: e => this.handleOnAfterOpen(e),
```

Comment:
needed to written this way to get correct `this` in `handleOnAfterOpen` function